src/environments/environment.prod.ts

```diff
@@ -1,3 +1,7 @@
 export const environment = {
-  production: true
-};
+  production: true,
+  NEW_API_URL: "https://altara-prod-api.herokuapp.com",

```

Comment:
lets use something like this for the .prod
`NEW_API_URL: process.env. NEW_API_URL === undefined ? 'https://altara-prod-api.herokuapp.com' : process.env. NEW_API_URL,`

Reply:
what of the `process.env` part?